bin/p2-exec/main.go

```diff
@@ -80,20 +81,55 @@ func main() {
 		}
 	}
 
+	cgroupPath := ""
+	if *podID != "" {
+		cgConfig, err := cgGetConfig("RESOURCE_LIMITS", "")
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		node, err := os.Hostname()
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		cg, err := cgroups.Find()
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		err = os.MkdirAll(filepath.Join(cg.CPU, "p2", node), 0755)
+		if err != nil && !os.IsExist(err) {
+			log.Fatal(err)
+		}
+		err = os.MkdirAll(filepath.Join(cg.Memory, "p2", node), 0755)
+		if err != nil && !os.IsExist(err) {
+			log.Fatal(err)
+		}
+		cgroupPath = "p2" + "/" + node
```

Comment:
`filepath.Join("p2",node)` is a nicer way to say this same thing. ditto on line 115